src/pages/SupplierRequisitionPage.js

```diff
@@ -41,6 +42,9 @@ export class SupplierRequisitionPage extends React.Component {
       modalKey: null,
       modalIsOpen: false,
       selection: [],
+      isProgramOrder: false,
+      orderType: null,
```

Comment:
isProgramOrder, orderType, thresholdMOS (as per below comments) should be set based on props.requisition

Reply:
- Removed `isProgramOrder` 4dab3938830fa467815b1fc17fbe120d8781c7a7
- Removed `orderType` state b582583b51aa8e56582f4c80a5ede306971861b0
- Using `ThresholdMOS` from requisition 9095693bb8516259926361a7bf6d4ca464bdad4f